garak/detectors/xss.py

```diff
@@ -44,10 +53,14 @@ def detect(self, attempt: garak.attempt.Attempt) -> List[float]:
             if output is None:
                 continue
 
-            found_exfil = True if re.search(_MARKDOWN_REGEX, output) else False
+            found_exfil = (
+                True if re.search(_MARKDOWN_IMG_REGEX, output, re.I) else False
+            )
 
             if found_exfil:
-                md_param_exfil_content = re.findall(_MARKDOWN_REGEX, output)[0]
+                md_param_exfil_content = re.findall(_MARKDOWN_IMG_REGEX, output, re.I)[
+                    3
+                ]
```

Comment:
Does this need a `len` check to be sure `[3]` exists?

Reply:
not if the stated expectations around L13 are met